js/data.js

```diff
@@ -436,7 +491,7 @@ var data = {
         kennung: [ITSE, HCT, SAMT],
         lehrform: ["Seminar", "Projekt"],
         modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
-        semester: ss14,
+        semester: [ss14, ss15],
```

Comment:
My guess:
It is not intended to be the same, looking at the naming scheme of Baudisch's seminars in the past.
I also think, Frau Pamperin only matches exact duplicates.